package.json

```diff
@@ -26,8 +27,12 @@
   "author": "Matteo Collina <hello@matteocollina.com>",
   "license": "MIT",
   "devDependencies": {
-    "pre-commit": "0.0.3",
+    "@types/node": "7.0.11",
     "chai": "~1.8.0",
-    "mocha": "~1.13.0"
-  }
+    "mocha": "~1.13.0",
+    "pre-commit": "0.0.3",
+    "typescript": "2.2.2"
+  },
+  "types": "index.d.ts",
+  "typings": "index.d.ts"
```

Comment:
Support for types has been introduced in TS 2.0, a lot of people still use https://github.com/typings/typings, this makes it compatible with both.